stencil.config.ts

```diff
@@ -1,17 +1,16 @@
-import { Config } from '@stencil/core';
+import { Config } from "@stencil/core";
 
 export const config: Config = {
-  namespace: 'shadowcat',
+  namespace: "shadowcat",
   outputTargets: [
     {
-      type: 'dist',
-      esmLoaderPath: '../loader'
```

Comment:
so I removed this. Not sure why it's there but it was different from the UI build so I removed it because I could figure out how to import the component otherwise